registration_form_with_db/form.py

```diff
@@ -0,0 +1,18 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, SubmitField
+from wtforms.validators import DataRequired, Email, Length, ValidationError
+from registration_form_with_db.model import User
+
+
+# добавлена проверка наличия/отсутствия пользователя с одинаковым email в нашей базе данных
+# поля с подтверждением пароля по условиям задачи нет, поэтому поле для сравнения пароля мы добавлять не будем
+class RegistrationForm(FlaskForm):
+    first_name = StringField('Имя', validators=[DataRequired()])
+    last_name = StringField('Фамилия', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Тут можно использовать `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```